code_benches/run.py

```diff
@@ -13,13 +17,19 @@
 # CSV file name
 CSV_NAME = "benchmarks.csv"
 
+# Make sure we're in the code_benches directory
+if os.path.dirname(sys.argv[0]):
+    os.chdir(os.path.dirname(sys.argv[0]))
 # Absolute path to the `code_benches` directory
 BENCH_ROOT = os.getcwd()
 # Absolute path to the `silverfish` directory
 ROOT_PATH = os.path.dirname(BENCH_ROOT)
 
 RUNTIME_PATH = ROOT_PATH + "/runtime"
-SILVERFISH_PATH = ROOT_PATH + "/target/release/silverfish"
+if "--release" in sys.argv:
+    SILVERFISH_PATH = ROOT_PATH + "/target/release/silverfish"
+else:
+    SILVERFISH_PATH = ROOT_PATH + "/target/debug/silverfish"
```

Comment:
I'm not sure if this is the right default. Maybe this is just my workflow, but I usually build silverfish in release mode, and this means more typing for me :P

Maybe this is too magic, but maybe we could use the newest of `target/release/silverfish` and  `target/debug/silverfish`?  (using whatever one is existant  if only one exists) Then add manual `--release` and `--debug` flags to override this magic choice? Let me know what you think on this--I'm happy to write that code as a follow up PR myself

Ignoring that, can you add validation that there are no useless args in argv? Don't want someone typoing `relese` or something like that

Reply:
Ah, I should have noted the default change.

I you run `cargo test` this will do the "right thing" based on if you passed --release or not.

I should be able to add a timestamp-based check (double-checking is that what you are asking for?), it will just take more perusing of Rust's APIs. The override flags make sense and should be easy to add in both the script and `cargo test`.

---

I can add the argv check. My default is to be the least invasive, but making it user-friendly is a good idea.